docs/build/micro-rdk/movement-sensor/gyro-mpu6050.md

```diff
@@ -117,7 +117,7 @@ Then remove and fill in the attributes as applicable to your movement sensor, ac
 | Name                  | Type    | Inclusion    | Description |
 | --------------------- | ------- | ------------ | ----------- |
 | `board`               | string  | **Required** | The `name` of the [board](/build/micro-rdk/board/) to which the device is wired. |
-| `i2c_bus`             | string  | **Required** | The `name` of the [I<sup>2</sup>C bus configured](/components/board/#i2cs) on your [board](/components/board/) wired to this device. |
+| `i2c_bus`             | string  | **Required** | The `name` of the I<sup>2</sup>C bus configured on your [board](/components/board/) wired to this device. |
```

Comment:
Nit: it's no longer configured on the board, and it's not really a name. I think you've used the word "index" instead of "name" elsewhere; I liked that, or just "The I2C bus wired to this device."